src/ValidationRules.jsx

```diff
@@ -33,7 +33,7 @@ const validations = {
 
     isNumber: value => validations.matchRegexp(value, /^-?[0-9]\d*(\d+)?$/i),
 
-    isFloat: value => validations.matchRegexp(value, /^(?:[1-9]\d*|0)?(?:\.\d+)?$/i),
+    isFloat: value => validations.matchRegexp(value, /^(?:-?[1-9]\d*|-?0)?(?:\.\d+)?$/i),
```

Comment:
first `-?` is ok, byt why is second one? to handle `-0`? does it make sense?

Reply:
Hi @NewOldMax, yes to handle -0, theoretically valid but happy to remove it from a practical perspective

Reply:
I've added the -0 back in, to allow for -0.X floats